src/main/kotlin/com/wire/bots/polls/dto/roman/Message.kt

```diff
@@ -97,7 +103,18 @@ data class Message(
          * Id of the button when it was clicked on.
          */
         val offset: Int?
-    )
+    ) {
+        override fun toString(): String {
+            return "PollObjectMessage(id='$id', buttons=$buttons, offset=$offset)"
+        }
+    }
+
+    /**
+     * Avoid printing out the token by mistake if object is printed.
+     */
+    override fun toString(): String {
+        return "Message(botId='$botId', userId=$userId, conversationId=$conversationId, type='$type', messageId=$messageId, text=$text, refMessageId=$refMessageId, reaction=$reaction, image=$image, handle=$handle, locale=$locale, poll=$poll, mimeType=$mimeType)"
```

Comment:
is this the default log format for this project? If not, I'd suggest to use one of the following formats in order to help with log parsing:

* [logfmt](https://brandur.org/logfmt)
* json

Reply:
All the log messages are already in JSON, this is configured project wise. Each log has a bunch of json fields from metadata, timestamp and the actual "message".
The messages are not json based, each logger can write any string they want. In this case, this is just the standard way objects are printed in Java